packer/linux/conf/buildkite-agent/systemd/buildkite-agent.service

```diff
@@ -24,4 +24,3 @@ KillMode=process
 
 [Install]
 WantedBy=multi-user.target
-DefaultInstance=1
```

Comment:
This is not needed as this unit is not used as a template and spits out a warning in journal